homeassistant/components/system_bridge/sensor.py

```diff
@@ -159,6 +205,7 @@ def memory_used(data: SystemBridgeCoordinatorData) -> float | None:
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfFrequency.GIGAHERTZ,
         device_class=SensorDeviceClass.FREQUENCY,
+        suggested_display_precision=2,
```

Comment:
It's a minor change, but maybe moving the rounds away could be a whole separate PR